src/app/shared/directives/default-image.directive.ts

```diff
@@ -45,5 +65,18 @@ export class DefaultImageDirective implements AfterViewInit, AfterViewChecked {
     }
    }
 
+   getBackGroundColor() {
```

Comment:
@preethijayanthi We'll need to set the background color in case of '?' as well, right?